openedx/core/djangoapps/oauth_dispatch/docs/decisions/0003-use-jwt-as-oauth-tokens-remove-openid-connect.rst

```diff
@@ -0,0 +1,154 @@
+3. Use JWT as OAuth2 Tokens; Remove OpenID Connect
+--------------------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The edX system has external OAuth2 client applications, including edX Mobile apps
+and external partner services. In addition, there are multiple edX microservices
+that are OAuth2 Clients of the LMS.
+
+Some of the internal microservice clients require `OpenID Connect`_ features.
+Specifically, they make use of the `ID Token`_ extension to get user profile
+details from the LMS via the OAuth protocol. The ID Token can also be forwarded
+from one microservice to another, allowing the recipient microservice to
+validate the identity of the token's owner without needing to reconnect with a
+centralized LMS.
+
+We have integrated our fork of DOP_ with support for OpenID Connect. So, an
+access_token request with a DOP client::
+
+    curl -X POST -d "client_id=abc&client_secret=def&grant_type=client_credentials" http://localhost:18000/oauth2/access_token/
+
+includes an id_token field::
+
+    {
+        "access_token": "xyz",
```

Comment:
In the below example you make it clear that this is a `<RANDOMLY-GENERATED-ACCESS-TOKEN>` you may want to do the same here for clarity.

Reply:
Good suggestion.  Updated.